examples/cargo_without_cmake/build.rs

```diff
@@ -8,20 +8,25 @@ use cxx_qt_build::CxxQtBuilder;
 
 fn main() {
     CxxQtBuilder::new()
-        // Link Qt's Qml and Network libraries.
-        // Qt Core is always linked and Qt Gui is linked by enabling the qt_gui Cargo feature.
+        // Link Qt's Network library
+        // - Qt Core is always linked
+        // - Qt Gui is linked by enabling the qt_gui Cargo feature.
+        // - Qt Qml is linked by enabling the qt_qml Cargo feature.
         .qt_module("Qml")
         .qt_module("Network")
         // Generate C++ from the `#[cxx_qt::bridge]` module
         .file("src/cxxqt_object.rs")
+        .file("src/qml.rs")
         // Generate C++ code from the .qrc file with the rcc tool
         // https://doc.qt.io/qt-6/resources.html
         .qrc("qml/qml.qrc")
         // Tell CxxQtBuilder's internal cc::Build struct to compile the manually
         // written C++ file in addition to the generated C++.
         .cc_builder(|cc| {
-            cc.file("cpp/run.cpp");
-            println!("cargo:rerun-if-changed=cpp/run.cpp");
+            // Include the cpp directory so CXX can find it
+            cc.include("cpp");
```

Comment:
```
  cargo:warning=/var/home/andrew/Projects/cxx-qt/target/debug/build/qml-minimal-no-cmake-b4bee99dc18dc06d/out/cxx-qt-gen/src/ffi.cxx.cpp:2:10: fatal error: register_types.cpp: No such file or directory
  cargo:warning=    2 | #include "register_types.cpp"
  cargo:warning=      |          ^~~~~~~~~~~~~~~~~~~~
```

Because there is `include!("register_types.cpp");` in the CXX bridge, that should probably actually be `register_types.h`

Reply:
Done

Reply:
Changing it to a header seems to have broken CI in fun ways :-D